site/content/docs/5.1/guidelines/modals.md

```diff
@@ -86,11 +86,13 @@ toc: true
           <button type="button" class="btn-close" data-bs-dismiss="modal"><span class="visually-hidden">Close</span></button>
         </div>
         <div class="modal-body" lang="zxx">
-          <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
-          <p>Consectetur lorem donec massa sapien faucibus et molestie. Nunc mattis enim ut tellus elementum sagittis vitae et leo.</p>
-          <p>Eleifend donec pretium vulputate sapien nec sagittis aliquam malesuada. Lorem sed risus ultricies tristique. Volutpat diam ut venenatis tellus in. Mauris vitae ultricies leo integer malesuada nunc vel risus.</p>
-          <p>Sit amet consectetur adipiscing elit duis tristique. Lacus laoreet non curabitur gravida arcu ac tortor. Pellentesque pulvinar pellentesque habitant morbi.</p>
-          <p>Adipiscing commodo elit at imperdiet dui accumsan sit amet. Sit amet aliquam id diam maecenas. Ornare arcu dui vivamus arcu felis bibendum ut tristique et. Tellus at urna condimentum mattis pellentesque id nibh tortor.</p>
+          <p>Please read through our Contributing guidelines.</p>
```

Comment:
I removed the Latin Lorem ipsum because it was also removed in some previous commits in Bootstrap. I didn't know what to write as a long text so I copied a part of the Boosted documentation 🤷